cmd/root.go

```diff
@@ -148,29 +173,29 @@ func Execute() {
 }
 
 func getConfigHome(opts *rootOptions) (string, error) {
+	// First check if the config path is provided via flags
 	if opts.configHome != "" {
-		output.Debugf("Using config directory from flag: %s", opts.configHome)
```

Comment:
These debug lines are useful in more verbose output (especially since e.g. an environment variable may make it hard to find where config is being read from).

Reply:
Correct! Adding them back!